Filelist

```diff
@@ -429,8 +431,13 @@ SRC_ALL =	\
 		src/xdiff/xutils.c \
 		src/xdiff/xutils.h \
 
+# Source files for all source archives.
+SRC_ALL =	\
+		$(FILE_GIT) \
+		$(BARE_SRC_ALL) \
```

Comment:
why split those?

Reply:
These same files are not directly source code.
For an offline local archive, they are, in principle, not required to build the program.